docs/modules/release-notes/pages/0.30.adoc

```diff
@@ -60,46 +143,99 @@ To learn more about this feature, consult https://github.com/apple/pkl-evolution
 === Trailing Commas
 
 Pkl's grammar has been updated to allow including a comma following the final item of comma-separated syntax elements (https://github.com/apple/pkl/pull/1137[#1137]).
+
 These syntax elements are affected:
 
-[source,pkl]
+[source%tested,pkl]
```

Comment:
We have tests called `DocSnippetTests` that will `pkl eval` each of these blocks if they have `source%tested`, and will simply parse them if they have `source%parsed`.